<launch>

  <!--打开CAN端口-->
  <node pkg="socketcan_bridge" type="socketcan_bridge_node" name="socketcan_bridge_node" />

  <!--接收MPC单片机CAN消息-->
  <node pkg="raw_data" type="ultrasonic_activate_node" name="ultrasonic_activate_node" >
    <rosparam command="load" file="$(find raw_data)/param/ultrasonic_raw.yaml" />
  </node>

  <!--超声波处理模块-->
  <node pkg="detection" type="ultrasonic_node" name="ultrasonic_node" output="screen" >
    <rosparam command="load" file="$(find detection)/param/ultrasonic_filter.yaml" />
  </node>

  <!--连接VLP16, 运行可行域分割程序-->
  <include file="$(find lidar_grid)/launch/lidar_grid.launch" />

  <!--rqt_plot-->
  <node pkg="rqt_plot" type="rqt_plot" name="rqt_plot1" />
  <node pkg="rqt_plot" type="rqt_plot" name="rqt_plot2" />

</launch>